python/cudf/cudf/core/column/numerical.py

```diff
@@ -573,14 +573,14 @@ def _find_value(
         found = 0
         if len(self):
             found = find(
-                self.data_array_view,
+                self.data_array_view(mode="read"),
                 value,
                 mask=self.mask,
             )
         if found == -1:
             if self.is_monotonic_increasing and closest:
                 found = find(
-                    self.data_array_view,
+                    self.data_array_view(mode="read"),
```

Comment:
Do we need to `acquire_spill_lock` around this?

Reply:
Yup, added 👍